src/transforms/transform-deployment.ts

```diff
@@ -290,7 +290,7 @@ export const transformDeployment = async (githubInstallationClient: GitHubInstal
 			schemaVersion: "1.0",
 			deploymentSequenceNumber: deployment.id,
 			updateSequenceNumber: deployment_status.id,
-			displayName: deployment.task,
+			displayName: message,
```

Comment:
Shouldn't we truncate the message to make sure we are not hitting API's limits?

Reply:
For the `pipeline.url`. What about the `log_url` ? 

It might be a breaking change, but we already use target_url on 'url' field when building this payload.